packages/react-components/react-storybook-addon/src/index.ts

```diff
@@ -3,3 +3,4 @@ export type { ThemeIds } from './theme';
 export { themes } from './theme';
 export { THEME_ID } from './constants';
 export { parameters } from './hooks';
+export { withFluentProvider } from './decorators/withFluentProvider';
```

Comment:
This is now being exported because during my test PRs with the swc based transpilation applied to the v9 packages, the VR tests that were written in CSF weren't working correctly. Initial findings are posted below (from https://github.com/microsoft/fluentui/pull/26504#issuecomment-1405366675)


> **Update**: Exporting `withFluentProvider` from `@fluentui/react-storybook-addon` and explicitly declaring this decorator within the v9 VR test package resolves the problem. Still not really sure why this needs to suddenly be explicitly declared though 😕
> 
> __________________________
> For some reason, having the packages transpiled with `SWC` is causing the v9 VR tests written in `CSF` to not render with the `FluentProvider`. It works fine when storybook starts locally but not when `build-storybook` is ran and then snapshots are taken. The issue stems from the way the stories are being compiled with the `build-storybook` because when those files were opened, the rendered component already doesn't have proper styling
> 
> ![image](https://user-images.githubusercontent.com/8649804/214909149-23a4f655-ddfb-4f86-ba25-fbeae0953785.png)

Reply:
hmm we should dig deeper. based on this public-docsite-v9 should be affected as well which is not AFAIK, right ?

Reply:
https://github.com/microsoft/fluentui/pull/27006

Reply:
I think we need to resolve this instead of using the workaround it sounds like a footgun especially because it's caused by something and we _**don't know why**_

Reply:
> #27006

@Hotell Just merged the changes in this PR with a test branch I have locally that has `swc` transpilation applied and it works like a charm! Screenshots are being generated with the addon styles once again without need to explicitly declare the decorator:

![image](https://user-images.githubusercontent.com/8649804/221910112-a8859465-5fca-4a5c-a5fb-7947028d5510.png)

Reply:
Thanks to @Hotell https://github.com/microsoft/fluentui/pull/27006 fixes this issue 🎉! I've removed the storybook-addon related changes from this PR